app/models/task.rb

```diff
@@ -0,0 +1,15 @@
+class Task < ApplicationRecord
+  belongs_to :project
+  validates :title, presence: true
+  validates :project, presence: true
+  validates :deadline, presence: true
```

Comment:
NBD, you can write it in single line, and project already validates in `belongs_to`
```suggestion
  validates :title, :deadline, presence: true
```